axelor-account/src/main/java/com/axelor/apps/account/web/JournalController.java

```diff
@@ -85,4 +85,26 @@ public void toggleStatus(ActionRequest request, ActionResponse response) {
       TraceBackService.trace(response, e);
     }
   }
+
+  public void changeConfig(ActionRequest request, ActionResponse response) {
+    try {
+      Journal journal = request.getContext().asType(Journal.class);
+
+      if (journal.getCompany().getAccountConfig() == null) {
+        response.setValue("authorizeSimulatedMove", false);
+        response.setValue("allowAccountingDaybook", false);
+      } else {
+        if (journal.getCompany().getAccountConfig().getIsActivateSimulatedMove() == null
+            || !journal.getCompany().getAccountConfig().getIsActivateSimulatedMove()) {
+          response.setValue("authorizeSimulatedMove", false);
```

Comment:
Is it possible to just do:
response.setValue("authorizeSimulatedMove", journal.getCompany().getAccountConfig().getIsActivateSimulatedMove())
?

Reply:
i can't do that if journal.getCompany().getAccountConfig() is null i will have a NPE